actix-web/src/middleware/err_handlers.rs

```diff
@@ -53,7 +55,38 @@ type ErrorHandler<B> = dyn Fn(ServiceResponse<B>) -> Result<ErrorHandlerResponse
 ///     .wrap(ErrorHandlers::new().handler(StatusCode::INTERNAL_SERVER_ERROR, add_error_header))
 ///     .service(web::resource("/").route(web::get().to(HttpResponse::InternalServerError)));
 /// ```
+/// ## Registering default handler
```

Comment:
Explain the precedence of overlapping handlers.

Reply:
hope the new docs are clearer. lmk if anything still doesn't make sense!